### DISCUSSION TITLE
How do you match on a literal exclamation point?

### DISCUSSION THREAD

**Discussion by nivekuil** [minad/consult#889] [Q&A] [answered]
isearch matches `!` literally, swiper can escape it as `\!`, but I can't figure out consult.

**Comment by minad** [comment, marked as answer]
Assuming that you use the Orderless default configuration, `=!` should do it. The trick here is that `=` is an affix style dispatcher, see https://github.com/oantolin/orderless#style-dispatchers. You can also set `orderless-style-dispatchers` to nil or customize `orderless-affix-dispatch-alist` if you prefer a different configuration.
